doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,71 @@ This Appendix has two goals:
 1. Provide guidance to VNF or more generally Application vendors on how to consume CNTT Reference Model and Architectures
 2. Provide usable definitions of maturity levels for VNF software architecture between Physical-to-Virtual migration and “Cloud Native”.
 
-The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform.
+The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform. So, it will primarily look at Operational impact (in steps like Install, Configure, Validate, Operate) towards full lifecycle automation of Develop decisions made (in steps like Design, Simulate, Build, Bundle).
```

Comment:
Sorry, I don't get the meaning of the new sentence. How come the developer decisions come in here?

Reply:
> Sorry, I don't get the meaning of the new sentence. How come the developer decisions come in here?

Agree. Reworded with ", after all Develop decisions were made".

Reply:
I get now what you mean, but I don't think we need to mention developer decisions. What about something like "towards full lifecycle automation supporting CI/CD"?

Reply:
> I get now what you mean, but I don't think we need to mention developer decisions. What about something like "towards full lifecycle automation supporting CI/CD"?

Thanks, this is good and I modified it that way now.